tools/unicode_case_conversion.py

```diff
@@ -395,7 +396,7 @@ def extract_character_pair_ranges(letter_case, reverse_letter_case):
             in_range = False
 
     # Remove all founded case mapping from the conversion tables after the scanning method
-    for idx in range(len(start_points)):
+    for idx, _ in enumerate(start_points):
```

Comment:
I see. Then it is also possible to remove the `letter_id = ..` line and use the `letter_id` instead of the `_`.